### ISSUE TITLE
Flowtypes not working

### ISSUE THREAD

**Issue by malekjaroslav** [jaredpalmer/razzle#358]
Hey guys, I am getting an error on Flowtype. I also added "flow" present to .babelrc but none of them works.

<img width="1331" alt="screen shot 2017-09-25 at 7 22 23 pm" src="https://user-images.githubusercontent.com/7753942/30808115-f2a688fe-a226-11e7-8122-613a09f2498c.png">

**Comment by malekjaroslav** [comment]
My fault, I was using `.jsx` files, but `babel-loader`'s regex in webpack config was only matching `.js`. This might be an issue as well. @jaredpalmer 

Fixed with: 
```
module.exports = {
    modify: (config, { target, dev }, webpack) => {
        config.module.rules[1].test = /\.(js|jsx)$/;
        config.module.rules[0].test = /\.(js|jsx)$/;
        return config;
    },
};

```

**Comment by jariz** [comment]
We fixed this already (859aa1341050b98178896c6321fd03359564765d) but we have yet to release a new version

**Comment by jaredpalmer** [comment]
Released
